nipype/pipeline/plugins/slurm.py

```diff
@@ -62,13 +62,17 @@ def __init__(self, **kwargs):
         super(SLURMPlugin, self).__init__(self._template, **kwargs)
 
     def _is_pending(self, taskid):
-        #  subprocess.Popen requires taskid to be a string
-        res = CommandLine(
-            'squeue',
-            args=' '.join(['-j', '%s' % taskid]),
-            resource_monitor=False,
-            terminal_output='allatonce').run()
-        return res.runtime.stdout.find(str(taskid)) > -1
+        try:
+            res = CommandLine(
+                'squeue',
+                args=' '.join(['-j', '%s' % taskid]),
+                resource_monitor=False,
+                terminal_output='allatonce').run()
+            return res.runtime.stdout.find(str(taskid)) > -1
+        except Exception as e:
```

Comment:
Based on https://github.com/nipy/nipype/pull/1853#issue-108145008, I think this should specifically be `RuntimeError`?

Otherwise, this looks reasonable.